sample-proposals/transferTokens.json

```diff
@@ -0,0 +1,7 @@
+{
+    "targets": ["0xf61E0C9d284508C284ba5d8a2eB2829581974E43"],
+    "values": ["0"],
+    "signatures": ["transfer(address,uint256)"],
+    "callDatas": ["0x000000000000000000000000d22506fbcb0fc301459ca8adddbd82c2895d1ccf00000000000000000000000000000000000000000000000000000000004c4b40"],
```

Comment:
it would be nice if we could do a follow up PR that allows you to provide the call data unencoded, and then we encode it in the JS script for you, based on the function signature, before sending.